include/zephyr/pm/policy.h

```diff
@@ -188,6 +189,9 @@ void pm_policy_latency_changed_unsubscribe(struct pm_policy_latency_subscription
  * will wake up the system at a known time in the future. By registering such
  * event, the policy manager will be able to decide whether certain power states
  * are worth entering or not.
+ * If the event needs to be handled with a very low latency constraint, calling this
```

Comment:
nit: "very" does not say much. I think it's better to rephrase to "no latency", or even better keep "low latency"
```suggestion
 * If the event needs to be handled with a low latency constraint, calling this
```